pager/victorops.go

```diff
@@ -56,17 +82,33 @@ func (v *VictorOps) LoadUsers(ctx context.Context) error {
 }
 
 func (v *VictorOps) LoadTeams(ctx context.Context) error {
-	vteams, _, err := v.client.GetAllTeams()
+	type Team struct {
+		Name          string `json:"name,omitempty"`
+		Slug          string `json:"slug,omitempty"`
+		IsDefaultTeam bool   `json:"isDefaultTeam,omitempty"`
+		SelfURL       string `json:"_selfUrl,omitempty"`
+	}
```

Comment:
They don't expose `SelfURL` in the SDK even when the value is returned by the API. At least they provide a way to access the underlying body, so I'm making one-off struct definition here (and in `LoadUsers` above) just to unmarshal and retrieve URL for annotations.